gtsam/gtsam.i

```diff
@@ -2759,21 +2759,29 @@ virtual class EssentialMatrixFactor : gtsam::NoiseModelFactor {
 };
 
 #include <gtsam/slam/dataset.h>
+
 class SfmTrack {
+  SfmTrack();
   Point3 point3() const;
   size_t number_measurements() const;
+  void set_point3(gtsam::Point3& p_);
```

Comment:
I'd rather see a new constructor than an imperative "set". "add" is fine as it is "part of the construction" of the object.